docs/source/usrman/env_var.rst

```diff
@@ -0,0 +1,52 @@
+.. _environment:
+
+Environment variables
+=====================
+
+Below we list the environment variables added since mpi4py v3.1.0. The environment
+variables overwrite their corresponding Python attributes at the ``MPI`` import time.
+
+``MPI4PY_RC_INITIALIZE``
```

Comment:
This is not the proper way. 

You should use 
```ReST
:envvar:`MPI4PY_RC_INITIALIZE`
```
for inline references. 

But for the listing of environment variables,  think you should typeset things like in [Python docs](https://github.com/python/cpython/blob/main/Doc/using/cmdline.rst#environment-variables)

Reply:
Thanks for pointer! I think `.. envvar::` is the proper usage. `:envvar:` is like you said for inline cross-reference, just like :mod:`mpi4py.MPI` or ":class:`mpi4py.MPI.Intercomm`" (dunno how to escape ticks on markdown...)